dashboard/.gitignore

```diff
@@ -24,3 +24,5 @@
 
 /public/code-studio-package
 /public/apps-package
+
+/db/schema_cache.dump
```

Comment:
I don't think this should be in `.gitignore`, we don't want/need this file to exist anywhere in any environment unless I'm mistaken?